Gifski/DraggableFile.swift

```diff
@@ -13,8 +13,17 @@ final class DraggableFile: NSImageView {
 		super.init(frame: frame)
 		isEditable = false
 		unregisterDraggedTypes()
+		
+		superview?.wantsLayer = true
+		wantsLayer = true
+		
+		let sh = NSShadow()
+		sh.shadowBlurRadius = 5.0
+		sh.shadowOffset = CGSize(width: 0, height: 0)
+		sh.shadowColor = NSColor(red: 0, green: 0, blue: 0, alpha: 0.5)
```

Comment:
The shadow looks good in light mode, but in dark mode it's almost invisible. Make this a named color in asset catalog and make the color in dark mode be `1` instead of `0.5`.